Documentation/artifact-format.md

```diff
@@ -127,7 +127,9 @@ Format: JSON
     {
       ...
     }
-  ]
+  ],
+  "compatibleDevices": ["vexpress-qemu", "beaglebone"],
+  "artifactId": "id"
```

Comment:
@maciejmrowiec: I think you said we should standardize on the naming here. Was it `device_types_compatible` and `artifact_id`? Please clarify.